buildTree.py

```diff
@@ -0,0 +1,32 @@
+Time complexity: O(n)
+Space complexity: O(n) for  result
```

Comment:
You don't generally include result in space complexity. The space will be used by the recursion stack.